admin/tests/unit/adapters/application-test.js

```diff
@@ -1,12 +0,0 @@
-import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
-
-module('Unit | Adapter | application', function(hooks) {
-  setupTest(hooks);
-
-  // Replace this with your real tests.
-  test('it exists', function(assert) {
-    const adapter = this.owner.lookup('adapter:application');
-    assert.ok(adapter);
-  });
-});
```

Comment:
Je suis un peu partagée d'enlever ces tests 'it exists' quand on a rien d'autre dans le fichier... 
En fait si on change d'endroit le fichier correspondant `application.js` , ou qu'on le supprime on a un test qui pète et c'est pas si mal je trouve. 
Surtout quand on fait du refacto, ça permet aussi de savoir où on en est et si on enlève le test rien nous empêche de bouger / renommer certains fichier et de tomber sur des erreurs qu'on ne comprends pas après . En plus ces tests unitaire sont peu couteux je pense, ça vaut le coup de les garder je pense.

Reply:
J'attends l'avis des autres. Très franchement ils n'apportent rien. Je pense que les autres tests (qui testent de vrais trucs) sont suffisants pour relever ce genre d'omission/suppression/oubli.

Reply:
retrait du commit incriminé, on verra plus tard.